queries/models/bilhetagem_interno/staging/schema.yml

```
@@ -0,0 +1,16 @@
+models:
+  - name: aux_disparos_whatsapp_caso_1_2
+    description: ""
+    columns:
+      - name: cpf_particao
+        description: "[partição]"
+        policy_tags:
+            - 'projects/rj-smtr-dev/locations/us/taxonomies/1688927184371506251/policyTags/8698618633874150807'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)